html/semantics/document-metadata/the-meta-element/pragma-directives/attr-meta-http-equiv-refresh/parsing.html

```diff
@@ -0,0 +1,103 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Parsing of meta refresh</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<style>
+iframe { display:none }
+</style>
+<body>
+<script>
+
+// failure to parse is []
+// success to parse is [time, url] where url is unresolved
+
+var tests_arr = [
+  {input: '', expected: []},
+  {input: '1', expected: [1, 'refresh.sub.html']},
+  {input: '1 ', expected: [1, 'refresh.sub.html']},
+  {input: '1\t', expected: [1, 'refresh.sub.html']},
+  {input: '1\r', expected: [1, 'refresh.sub.html']},
+  {input: '1\n', expected: [1, 'refresh.sub.html']},
+  {input: '1\f', expected: [1, 'refresh.sub.html']},
+  {input: '1;', expected: [1, 'refresh.sub.html']},
+  {input: '1,', expected: [1, 'refresh.sub.html']},
+  {input: '1; url=foo', expected: [1, 'foo']},
+  {input: '1, url=foo', expected: [1, 'foo']},
+  {input: '1 url=foo', expected: [1, 'foo']},
+  {input: '1;\turl=foo', expected: [1, 'foo']},
```

Comment:
Oooh, nice, I didn't think to test \t and friends. What about multiple spaces, e.g. `1;      url=foo`? Or `1,url      = foo` or `1 url    =    foo`?